ntpcheck/cmd/diag.go

```diff
@@ -122,6 +123,18 @@ func checkJitter(r *checker.NTPCheckResult) (status, string) {
 	)
 }
 
+func checkCorrectionMetric(r *checker.NTPCheckResult) (status, string) {
+	const warnThreshold time.Duration = time.Minute
+	const failThreshold time.Duration = 10 * time.Minute
+	return checkAgainstThreshold(
+		"Current correction",
+		r.Correction,
```

Comment:
This value is in seconds, so should be converted to ms for `checkAgainstThreshold`